src/libstore/filetransfer.cc

```diff
@@ -72,6 +72,8 @@ struct curlFileTransfer : public FileTransfer
 
         curl_off_t writtenToSink = 0;
 
+        inline static const std::set<long> successfulStatuses;
```

Comment:
This looks like it lacks an initializer?

Reply:
Thanks, fixed